app/components/simple_format_component.rb

```diff
@@ -0,0 +1,41 @@
+class SimpleFormatComponent < ApplicationComponent
+  attr_reader :chunks
+
+  def initialize(text, allow_a: true)
+    @text = Anchored::Linker.auto_link(text, target: '_blank', rel: 'noopener')
```

Comment:
```suggestion
    @text = Anchored::Linker.auto_link(text, **h.external_link_attributes)
```
(pour rajouter aussi le noreferrer vu qu'on sait pas vers qui on renvoie)